app/controllers/exchanges/hbx_profiles_controller.rb

```diff
@@ -517,8 +519,16 @@ def carrier
 
     @products_data = products.map { |product| product_data(product) }
 
+    @filter_options = {
+      plan_types: { hmo: 'HMO', ppo: 'PPO', epo: 'EPO', pos: 'POS', pvp: 'PVP', standard: 'Standard' },
+      rating_areas: pvp_rating_area_options(products),
+      metal_levels: products.map { |p| [p.metal_level_kind, p.metal_level_kind.to_s.capitalize] }.uniq.to_h,
+      networks: { metro: "Metro", nationwide: "Nationwide", '3': "3", '4': 4}
```

Comment:
this hardcoded values looks strange, is this something that could change in the future ? should we think on creating an environment variable ? or a configuration setting ?

Reply:
I will put it into config variable. Thank you